src/main/java/org/jabref/gui/EntryTypeViewModel.java

```diff
@@ -176,6 +176,10 @@ public void runFetcherWorker() {
                 searchSuccesfulProperty.set(true);
             } else if (StringUtil.isBlank(idText.getValue())) {
                 dialogService.showWarningDialogAndWait(Localization.lang("Empty search ID"), Localization.lang("The given search ID was empty."));
+            } else if (result.isEmpty()) {
+                String fetcher = selectedItemProperty().getValue().getName();
+                String searchId = idText.getValue();
+                dialogService.showErrorDialogAndWait(Localization.lang("Fetcher '%0' did not find an entry for id '%1'.", fetcher, searchId));
```

Comment:
Can I leave this error message or should it be unique?

Reply:
Yes, looks good so far